elyra/pipeline/handlers.py

```diff
@@ -96,21 +98,47 @@ async def post(self, *args, **kwargs):
         self.finish(json_msg)
 
 
-class PipelineConfigHandler(HttpErrorMixin, APIHandler):
-    """Handler to expose method calls to retrieve pipelines editor configuration"""
+class PipelineComponentHandler(HttpErrorMixin, APIHandler):
+    """Handler to expose method calls to retrieve pipelines editor component configuration"""
 
-    valid_resources = ["palette", "properties"]
+    valid_processors = ["local", "kfp", "airflow"]
```

Comment:
We should expose a `get_processor_types() -> List` method on `PipelineProcessorRegistry` from which these are derived.  This way it's always based on the set of "registered" processors.

The implementation should be as simple as:
```python
def get_processor_types() -> List:
    return self._processors.keys()
```